src/content/en/2020/media.md

```diff
@@ -4,13 +4,14 @@ part_number: I
 chapter_number: 5
 title: Media
 description: Media chapter of the 2020 Web Almanac covering image file sizes and formats, responsive images, client hints, lazy loading, accessibility and video.
-authors: [tpiros, bseymour]
-reviewers: [nhoizey, zcorpan, colinbendell, dougsillars]
+authors: [tpiros, bseymour, eeeps]
+reviewers: [nhoizey, colinbendell, dougsillars]
```

Comment:
@tpiros can you confirm these are correct contributors in rough order of contribution?

Reply:
Confirmed and added the person missing